persistence/gov.go

```diff
@@ -35,6 +40,34 @@ func (p PostgresContext) InitParams() error {
 	return err
 }
 
+func (p PostgresContext) GetParameter(paramName string, height int64) (v any, err error) {
```

Comment:
This business logic is very proto-specific and how we configured the proto as well. Can you add a comment explaining what's happening? A condensed version of what you have in the github description should be good enough.

Reply:
Added code explaining the `init()` function parsing the `Params{}` struct to build the in memory map and a shorter comment for the `GetParameter()` function now the logic is seperated